### PR TITLE
feat: add angular example and fix issues for examples

### PR DESCRIPTION
**Pull request by nmerget**

### CODE REVIEW

File: examples/angular-example/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+indent_size = 2
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.ts]
+quote_type = single
+
+[*.md]
+max_line_length = off
+trim_trailing_whitespace = false
```
Comment (mfranzke) [inline examples/angular-example/.editorconfig:16]:
```suggestion
[*]
indent_style = space
indent_size = 2

[*.md]
max_line_length = off
trim_trailing_whitespace = false
```

File: examples/angular-example/.vscode/extensions.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=827846
```
Comment (mfranzke) [inline examples/angular-example/.vscode/extensions.json:2]: Comments aren't allowed this way in JSON. Is it possible to use either one of these two solutions instead?
- [`JSON5`](https://json5.org/)
- using a hack to define this like this:
```suggestion
  "_for_more_information_visit": "https://go.microsoft.com/fwlink/?linkid=827846"
```

File: examples/angular-example/.vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```
Comment (mfranzke) [inline examples/angular-example/.vscode/launch.json:2]: Comments aren't allowed this way in JSON. Is it possible to use either one of these two solutions instead?
- [`JSON5`](https://json5.org/)
- using a hack to define this like this (does the functionality still work in VS Code?):
```suggestion
  "_for_more_information_visit": "https://go.microsoft.com/fwlink/?linkid=830387"
```

File: examples/angular-example/.vscode/tasks.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
+{
+  // For more information, visit: https://go.microsoft.com/fwlink/?LinkId=733558
```
Comment (mfranzke) [inline examples/angular-example/.vscode/tasks.json:2]: Comments aren't allowed this way in JSON. Is it possible to use either one of these two solutions instead?
- [`JSON5`](https://json5.org/)
- using a hack to define this like this (does the functionality still work in VS Code?):
```suggestion
  "_for_more_information_visit": "https://go.microsoft.com/fwlink/?LinkId=733558"
```

File: examples/angular-example/.editorconfig (hunk 1 above)
Comment (mfranzke) [reply examples/angular-example/.editorconfig:16]: As these are self-enclosing example files, there shouldn't be any reliance on files outside of these `examples` path. So this comment is most likely obsolete, right?

File: examples/angular-example/tsconfig.app.json [hunk 5]
```diff
@@ -0,0 +1,10 @@
+/* To learn more about this file see: https://angular.io/config/tsconfig. */
+{
```
Comment (mfranzke) [inline examples/angular-example/tsconfig.app.json:2]: Comments aren't allowed this way in JSON. Is it possible to use either one of these two solutions instead?
- [`JSON5`](https://json5.org/)
- using a hack to define this like this (does the functionality still work in VS Code?):

```suggestion
{
  "_to_learn_more_about_this_file_see": "https://angular.io/config/tsconfig",
```

File: examples/angular-example/tsconfig.json [hunk 6]
```diff
@@ -0,0 +1,29 @@
+/* To learn more about this file see: https://angular.io/config/tsconfig. */
+{
```
Comment (mfranzke) [inline examples/angular-example/tsconfig.json:2]: Comments most likely aren't allowed this way in JSON. Is it possible to use either one of these two solutions instead?
- [`JSON5`](https://json5.org/)
- Creating another file at the same spot with the same name, but the fileending `.md`
- using a hack to define this like this (does the functionality still work?):

```suggestion
{
  "_to_learn_more_about_this_file_see": "https://angular.io/config/tsconfig",
```

**Review by mfranzke — APPROVED** [review]

File: examples/angular-example/.editorconfig (hunk 1 above)
Comment (nmerget) [reply examples/angular-example/.editorconfig:16]: Yes your are right. Those examples should be used as starting point for a standalone app

Comment (mfranzke) [reply examples/angular-example/.editorconfig:16]: Than it wasn't necessary to commit my suggestion. I've reverted that.

**Review by mfranzke — APPROVED** [review]

**Pull request merged**